package.json

```diff
@@ -26,8 +25,8 @@
     "serve": "node ./dev-utils/run-script.js startTestServers",
     "coverage": "lerna run build:module && lerna run karma:coverage",
     "bench:dev": "lerna run build:module && node ./scripts/benchmarks.js",
-    "package:snapshot": "npx lerna exec npm pack",
-    "clean": "npx lerna exec -- rm -rf dist/",
+    "package:snapshot": "lerna exec npm pack",
+    "clean": "lerna exec -- rm -rf dist/",
```

Comment:
this command is not currently used in CI, that's why I have not modified any yaml file adding the "npm ci"